grpc-server/workflow.go

```diff
@@ -270,15 +270,18 @@ func (s *server) ShowWorkflowEvents(req *workflow.GetRequest, stream workflow.Wo
 	return nil
```

Comment:
I think adding createYaml to the commit message would help when looking through git history:

> createYaml: use more descriptive argument names

Rename temp and tempData to templateID and templateData for better clarity.